.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: go
 go:
- - 1.7
```

Comment:
(sorry for sitting on this).

If we're preserving compatibility with 1.7, it seems best to leave this as 1.7 so that we can check that. If the project builds ok on 1.7 there's a good chance it will also build ok on 1.12. The reverse is not true.